db/migrate/20171130032001_remove_subscribed_from_journal.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSubscribedFromJournal < ActiveRecord::Migration
+  def change
+    remove_column :journals, :subscribed, :boolean
```

Comment:
Good job cleaning this up - it's easy to forget to do this and 6 months from now you'll forget whether it is being used or not.